pyethapp/config.py

```diff
@@ -68,6 +68,18 @@ def check_config(config, required_config=required_config):
     return True
 
 
+def validate_alt_config_file(ctx, param, value):
+    if value:
```

Comment:
can you please write documentation for the function (and best whenever you touch code).